src/modality-specific-files/task-events.md

```diff
@@ -104,8 +104,15 @@ In the accompanying JSON sidecar, the `trial_type` column might look as follows:
         "Description": "Channel(s) associated with the event",
         "Delimiter": "|"
     },
-    "artifact": {
-        "Description": "Description of an artifact."
+    "annots": {
+        "LongName": "Annotations",
+        "Description": "Hierarchical Event Descriptors (HED) annotations",
```

Comment:
Change to:
"Description": "Annotations associated with specific channels",

Reply:
```suggestion
        "Description": "Annotations associated with channels indicated in the channel column.",
```